### PR TITLE
Changing login to oauth2

### PR DESCRIPTION
**Pull request by djbielejeski**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -92,7 +92,11 @@
     "why-did-you-update": "1.0.6",
     "winston": "2.1.1"
   },
+  "optionalDependencies": {
+    "win-node-env": "0.4.0"
+  },
   "scripts": {
+    "start-nodemon": "nodemon app.js",
```
Comment (xjpro) [inline package.json]: what dis

File: server/config/auth.js [hunk 2]
```diff
@@ -38,22 +36,25 @@ auth.init = function(app) {
...
-	function loginCallback(tokens, profile, done) {
-		userService.findOrCreateBunkerUser(profile).nodeify(done);
-	}
+	app.get("/login/google", passport.authenticate("google"));
 
-	app.post("/auth/googleCallback", passport.authenticate("google"), function(req, res) {
+	app.get("/auth/googleReturn", passport.authenticate("google"), function(req, res) {
 		req.session.googleCredentials = req.authInfo;
-		res.json({});
+		//res.json(req.query);
```
Comment (xjpro) [inline server/config/auth.js]: begone old line of code

File: package.json (hunk 1 above)
Comment (djbielejeski) [reply package.json]: https://nodemon.io/

Nodemon is a utility that will monitor for any changes in your source and automatically restart your server. Perfect for development

Comment (xjpro) [reply package.json]: 👍

**Pull request merged**